### PR TITLE
Use new review based pricing everywhere

### PR DESCRIPTION
**Pull request by salbertson**
Pricing on landing page
Screen Shot 2021-01-28 at 8 07 48 PM

Pricing on Account page
Screen Shot 2021-01-28 at 8 07 33 PM

### CODE REVIEW

File: app/services/repo_subscriber.rb [hunk 1]
```diff
@@ -30,8 +30,14 @@ def unsubscribe
   def create_subscription
     plan_selector = PlanSelector.new(user: user, repo: repo)
 
+    plan = if plan_selector.current_plan.open_source?
+      plan_selector.next_plan
```
[the remark this answers is not in the document]
Comment (gylaz) [reply app/services/repo_subscriber.rb:34]: Maybe pull this into a method to avoid these warning?

File: app/views/home/_pricing.html.erb [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
-          <p class="c-plan__price"><span class="c-plan__dollar-amount">$249</span> month</p>
+          <p class="c-plan__title">Husky</p>
+          <p class="c-plan__allowance">Up to <strong>3,000 private</strong> reviews</p>
+          <p class="c-plan__price"><span class="c-plan__dollar-amount">$199</span> month</p>
         </li>
+        <!--li class="o-grid__item o-grid__item--span-3@medium c-plan c-plan--private">
+          <p class="c-plan__title">Great Dane</p>
+          <p class="c-plan__allowance">More than <strong>3,000 private</strong> reviews</p>
+          <p class="c-plan__price"><span class="c-plan__dollar-amount">$299</span> month</p>
+        </li-->
```
Comment (gylaz) [inline app/views/home/_pricing.html.erb:37]: Delete this or what?

File: spec/features/account_spec.rb [hunk 3]
```diff
@@ -27,32 +27,44 @@
...
+      expect(find(".plan-allowance")).to have_text "Up to 3,000 Reviews"
+      expect(find(".plan-price")).to have_text "$199 month"
+    end
+
+    within(plans[5]) do
       expect(find(".plan-title")).to have_text "Great Dane"
-      expect(find(".plan-allowance")).to have_text "Up to 30 Repos"
-      expect(find(".plan-price")).to have_text "$249 month"
+      expect(find(".plan-allowance")).to have_text "Up to 10,000 Reviews"
+      expect(find(".plan-price")).to have_text "$299 month"
```
Comment (gylaz) [inline spec/features/account_spec.rb:67]: Do we advertize this plan?

**Review by gylaz — APPROVED** [review]

File: app/views/home/_pricing.html.erb (hunk 2 above)
Comment (salbertson) [reply app/views/home/_pricing.html.erb:37]: I was going to leave it and advertise this plan once we get the CSS right.

File: spec/features/account_spec.rb (hunk 3 above)
Comment (salbertson) [reply spec/features/account_spec.rb:67]: It's shown on the Account page, just not on the landing page yet.

**Pull request merged**